### ISSUE TITLE
KMG Capture: vJoy support for accessibility purposes

### ISSUE THREAD

**Issue by J2Kbr** [J2Kbr/GtunerIV#303]
**oneswitch** wrote:

> KMG Capture currently gives access to keyboard, mouse and game-pads.
> 
> ...Gamepads does not seem to pick up unusual controls. I would especially like for it to pick-up on "vJoy" (a virtual joystick: http://vjoystick.sourceforge.net/joomla/ (and this is the version I'm using: https://www.dropbox.com/sh/0jb45pqtpi51c9g/AABrRXsOLpNIEpD30uXe9P7fa?dl=0).
> 
> If KMG could pick-up upon vJoy and allow for keystrokes to be passed through to other software, I believe I'd be able to get the OneSwitch Pulse system working, that people like Colin McDonnell use here:
> 
> https://switchgaming.blogspot.com/2015/05/one-switch-call-of-duty.html
> 
> Fingers crossed. :) Thanks for all the brilliant work on improving the Titan Two.

**Comment by oneswitch** [comment]
We've cracked this as best we can, thanks Jefferson. 

We're using a mix of Input Translator detection, and scripts to pick up the analogue controls. We're right on the cusp of 100% T2 code use, so Graham's done amazing work to squeeze it in. Hope to get a video up next week showing it working with speech control on consoles alongside the Game Control Mixer.

The controls are different to the One Switch Pulse system I used with Colin, but for that the T1 works well.

Many thanks again.
